include/core/parameters_lagrangian.h

```diff
@@ -231,6 +231,39 @@ namespace Parameters
       parse_parameters(ParameterHandler &prm);
     };
 
+    /**
+     * @brief ForceTorqueOnWall - Defines the parameters for the
+     * force and torques calculation on boundaries of the domain.
+     */
+    template <int dim>
+    class ForceTorqueOnWall
+    {
+    public:
+      // Enable force post-processing
+      bool calculate_force_torque;
+
+      // Choosing how the outputs is gonna be displayed
+      enum class Verbosity
```

Comment:
The Verbosity enum already exists in parameter. Why recode it?

Reply:
I don't wanna have a verbosity dependence. I mean that i wanna be verbose for force and torque and not for other parameters.

Reply:
No what I meant is that enum already exists elsewhere. There is no need to recreate the enum, you can use it as it is.
Look here : https://github.com/lethe-cfd/lethe/blob/master/include/core/parameters.h#L46
It already exists. Use it...

Reply:
perfect